### ISSUE TITLE
Issue with compiling LinkGenerator for windows using newest LinkGrammar vers 5.10.3

### ISSUE THREAD

**Issue by DennGru** [opencog/link-grammar#1284]
Can't compile the latest version 5.10.3  LinkGenerator using MS Visual Studio.
LinkGenerator appears in the .sln file, but LinkGenerator.vcxproj  is missing in the msvc folder.
I checked and the previous version of this missing file contains mention of getopt and needs modification to suit the new version.
Just an omission from the tarball?
Dennis in Tasmania

**Comment by ampli** [comment]
It was omitted in error - will appear in the tarball of the next version.
Meanwhile please download it from GitHub.

**Comment by DennGru** [comment]
Thanks again Amir,
It works now once I add the getopt folder AND the LinkGenerator.vcxproj from the previous version.

**Comment by ampli** [comment]
Hi Dennis,
Thanks for pointing out the `getopt` omission. I have just added it to the fix.

**Comment by ampli** [comment]
@linas,
The word **build** has no noun definition in the dict.
I have a list of such words for which one POS is missing.

```
linkparser> Fix a build break
No complete linkages found.
Found 2 linkages (2 had no P.P. violations) at null count 1
	Linkage 1, cost vector = (UNUSED=1 DIS= 0.00 LEN=7)

            +-------Os-------+
    +-->Wg--+   +----Ds**c---+
    |       |   |            |
LEFT-WALL fix.v a [build] break.n
```

**Comment by linas** [comment]
@ampli send me the list, or place it in a new issue.

**Comment by linas** [comment]
Closing, the fix is available in 5.10.4 which was released today.
